File: beginner/palindrom.py
def is_palindrom(word) -> bool : 
    # if word[:] == word[::-1]:
    #     return True
    # else:
    #     return False
    for i in range(len(word)//2) : 
        print(i)
        if word[i] == word[len(word)-i-1] : 
            print('first : ', word[i])
            print('last : ',word[len(word)-i-1])
            continue
        else : 
            return False
    else:
        return True

word = input('enter word : ')
print(is_palindrom(word))

File: beginner/js/palindrom.js
const prompt = require('prompt-sync')();  // ← import prompt-sync

function is_palindrom(word){
    console.log('word is : ', word)
    for (let i=0; i < Math.floor(word.length) /2; i++){
        console.log(i)
        if (word[i] !== word[word.length - i - 1]) {
                return false;
            }
        }
    return true;
}

const word = prompt("enter word : ")
if (is_palindrom(word)){
    console.log('yes palindrom')
}
else{
    console.log('false')
}